src/CMakeLists.txt

```diff
@@ -151,14 +151,17 @@ if(NOT UMF_DISABLE_HWLOC)
     set(UMF_LIBS ${UMF_LIBS} ${LIBHWLOC_LIBRARIES})
 
     if(NOT WINDOWS)
-        add_optional_symbol(umfMemspaceCreateFromNumaArray)
         add_optional_symbol(umfMemspaceHighestBandwidthGet)
         add_optional_symbol(umfMemspaceHighestCapacityGet)
         add_optional_symbol(umfMemspaceHostAllGet)
         add_optional_symbol(umfMemspaceLowestLatencyGet)
     endif()
 endif()
 
+if(UMF_DISABLE_HWLOC OR WINDOWS)
+    set(UMF_SOURCES ${UMF_SOURCES} memspaces/memspace_unsupported_numa.c)
```

Comment:
memspace_numa_unsupported.c 

It matter as files are sorted alphabetically, so you we have both versions of the file next to each other.

Reply:
done